tox.ini

```diff
@@ -12,20 +12,29 @@ commands =
     coverage report
     coverage xml
 
+[testenv:isal]
+deps =
+    pytest
+    coverage
+    isal
+
 [testenv:flake8]
 basepython = python3.6
 deps = flake8
 commands = flake8 src/ tests/
+skip_install = true
 
 [testenv:mypy]
 basepython = python3.6
 deps = mypy
-commands = mypy src/
+commands = mypy --ignore-missing-imports src/
```

Comment:
Using this command-line option is only recommended as [a last resort](https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports). You should be able to just add a `# type: ignore` to the `from isal import ...` line instead.

Reply:
Ah. Thanks! That is good to know. This missing import error is pretty annoying when using mypy and it is nice to know the proper fix.

Reply:
Done